app/invocation/invocation_action_card.tsx

```diff
@@ -0,0 +1,64 @@
+import pako from "pako";
+import React from "react";
+import format from "../format/format";
+import InvocationModel from "./invocation_model";
+import rpcService from "../service/rpc_service";
+
+interface Props {
+  model: InvocationModel;
+  inProgress: boolean;
+  hash: string;
+  size: Long;
+}
+
+interface State {
+  loading: boolean;
+  contents: string;
+}
+
+export default class ActionCardComponent extends React.Component {
+  props: Props;
+
+  state: State = {
+    loading: false,
+    contents: ""
+  };
+
+  fetchAction() {
+    let actionFile = "bytestream://localhost:1987/blobs/" + this.props.hash + "/" + String(this.props.size);
```

Comment:
nit: add a TODO to replace `localhost:1987` with the BB server URI, and also prefix the path with the instance name